src/components/input-field/examples/input-field-selection.tsx

```
@@ -0,0 +1,78 @@
+            <limel-input-field
+                label="Text field"
+                value={this.value}
+                onChange={this.handleChange}
+                ref={this.getInputFieldRef}
+                tabindex="0"
+                style={{ 'margin-bottom': '1rem' }}
+            />,
+            <limel-button
+                onClick={this.insertTextAtCursor}
+                label="Insert '[INSERTED]' at cursor"
+            />,
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)